### ISSUE TITLE
Method image file not transferred from .net

### ISSUE THREAD

**Issue by scottofletcher** [participedia/frontend#842]
Lead image in .net methods entry broken on XYZ

.net:
![image](https://user-images.githubusercontent.com/29512491/41423161-6f961194-6faf-11e8-9df8-da5aa9796b72.png)

XYZ from search page:
![image](https://user-images.githubusercontent.com/29512491/41423191-85794454-6faf-11e8-81e6-b471c431a55b.png)

XYZ on narrative view:
![image](https://user-images.githubusercontent.com/29512491/41423214-93c34a28-6faf-11e8-918e-0da8901dfd70.png)

**Comment by scottofletcher** [comment]
I would manually fix it but unfortunately the .net image file is the size of my thumb nail (that was a bug too - if you upload your image it got shrunk; if you pasted it into the narrative text, it's size wouldn't change)

**Comment by jesicarson** [comment]
Yep, .net lead images are often small and pixelated. Some are also broken on import (not sure why, thats the bug). In this case, I've saved the .net one and replaced the broken one on .xyz with it. Not ideal, but maybe needs a new image anyway. Also could be related to https://github.com/participedia/frontend/issues/302

![screen shot 2018-06-14 at 11 02 54 am](https://user-images.githubusercontent.com/13560530/41429698-af75bf40-6fc2-11e8-8128-f906e5f1384f.png)

**Comment by scottofletcher** [comment]
Ok, yeah not super ideal since we have a few methods written by their designers w/original photos (ie. not available anywhere else). Let me know if this is a bug we can fix or whether I should start contacting some of those authors (eg. Tara Mahoney). 

Also, weren't we going to make it easier to display images in-text? What ever happened to that? I've had a few authors with entries that need graphs/tables/images mid-narrative

**Comment by jesicarson** [comment]
@scottofletcher just had a breeze through methods on .net and it looks to me that only a handful have images to begin with. And yes, the ones that did have images on .net are all broken on .xyz (this is a bug). However, since there are a small number of methods with images, I'd recommend (if you like) to manually replace them and contact authors if needed to get originals if they can't be saved from .net. Its going to be a while before dethe gets to this, so your call if you want to tackle it manually or not.

**Comment by scottofletcher** [comment]
@jesicarson yeah no worries, I'll wait and see if Dethe can find a fix before I start contacting authors. although there aren't that many with images, those are probably the most valuable images on the site because they were actually taken by the authors, most of whom developed the method. anyways, we'll cross that bridge when it comes
